packages/typescript-generator/templates/typescript-package/package.json.hbs

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@pagopa/{{packageName}}",
+  "version": "0.1.0",
+  "type": "module",
+  "description": "{{description}}",
+  "repository": {
+    "type": "git",
+
+    "url": "git+https://github.com/pagopa/dx.git",
+    "directory": "packages/{{packageName}}"
+  },
+  "keywords": [],
+  "files": [ "dist" ],
+  "main": "./dist/index.js",
+  "exports": {
+    ".": { "import": "./dist/index.js" }
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@pagopa/eslint-config": "workspace:^",
+    "@tsconfig/node20": "^20.1.5",
+    "@types/node": "^20.12.2",
+    "eslint": "8.57.0",
+    "tsup": "^8.5.0",
+    "typescript": "~5.8.3",
+    "vitest": "^1.6.0"
+  },
+  "scripts": {
+    "start": "node .",
+    "build": "tsup",
+    "dev": "tsup --watch",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint --fix src",
```

Comment:
I would make the setup as easier as possible - IMHO there are no reason for pushing to our consumers a code bundler. `tsc` is easy to use and it is good enough for the majority of use case we have.

(I would include `esbuild/tsup` only for advanced setup)

```suggestion
    "build": "tsc",
    "build:watch": "tsc --watch",
    "typecheck": "tsc --noEmit",
    "lint": "eslint --fix src",
```

Reply:
@kin0992 yes for libraries I think is the best option, maybe we can try to use `esbuild` directly (`tsup` is just a thin wrapper of it) and add the latter to the technology radar